config/currency_non_iso.yml

```diff
@@ -94,3 +94,18 @@ gbx:
   thousands_separator: ","
   iso_numeric: ''
   smallest_denomination: 1
+usdc:
```

Comment:
Contents of this file was taken from: https://github.com/RubyMoney/money/blob/main/config/currency_non_iso.json#L130